.github/workflows/deploy.yml

```diff
@@ -56,7 +55,8 @@ jobs:
           echo "ORIGIN=${owner}/${repository}" >> ${GITHUB_ENV}
           echo "TARGET_BRANCH=gh-pages" >> ${GITHUB_ENV}
           # "url:" is for the opengraph tags, and it can't be relative
-          echo "url: https://${owner}.github.io/${repository}" >> _extra_config.yml
+          echo "url: https://${owner}.github.io" >> _extra_config.yml
+          echo "baseurl: /${repository}" >> _extra_config.yml
```

Comment:
We have `JEKYLL_BASE_URL` for it: https://github.com/apache/arrow-site/blob/90107a822a97f69da6b1999fd44bd9dfa2ab8bee/Rakefile#L54-L55

Why do we need this?

Reply:
Thanks. Updated.  Preview at: 
https://bkmgit.github.io/arrow-site/
due to
https://github.com/bkmgit/arrow-site/pull/4